p2p/docs/trustmetric.md

```diff
@@ -0,0 +1,16 @@
+
+The trust metric tracks the quality of the peers.
```

Comment:
Additional detail could be provided in this file using information from the architecture documents if you wish.